src/components/today-news/ai-summary-section.tsx

```diff
@@ -0,0 +1,17 @@
+import { Tooltip } from '../ui/tooltip';
+import { Text } from '@radix-ui/themes';
+
+interface AiSummarySectionProps {
+  description: string;
+}
```

Comment:
export 하는게 아니기 때문에 Prop 로 이름을 지어도 될 것 같아요~
```suggestion
interface Props {
  description: string;
}
```

Reply:
[refactor: 코드리뷰 반영 AiSummarySectionProps -> Props](https://github.com/anyway-team/frontend/pull/1/commits/6f97d6d08fe487e9ff9536eb436373a663e871fd)